apps/tlon-mobile/src/fixtures/Channel.fixture.tsx

```diff
@@ -362,6 +362,7 @@ function ChannelWithControlledPostLoading() {
             channel: baseProps.channel,
             post: anchorPost,
           }),
+          hasNewerPosts: true,
```

Comment:
without this line, this fixture would continuously load new posts, scroll to bottom, load more posts, scroll to bottom, etc... I'm guessing #3893 made the bug apparent